frontend/src/pages/dashboard.tsx

```diff
@@ -1,80 +1,92 @@
 import callApi from "../utils/callApi";
 import LeaderboardView from "../views/LeaderboardView";
 import Button from "../components/Button";
 import Container from "../components/Container";
 import Theme from "../components/Theme";
-import {mobile} from "../Global";
-import {useThemeStore} from "../store/themeStore";
+import { mobile } from "../Global";
+import { useThemeStore } from "../store/themeStore";
 
 const Dashboard = (): JSX.Element => {
+  const navigate = useNavigate();
 
-    const navigate = useNavigate();
+  const [auth, setAuth] = useState(false);
 
-    const [auth, setAuth] = useState(false);
+  useEffect(() => {
+    callApi("GET", "/api/home", null).then((res) => {
+      if (res.ok) {
+        setAuth(true);
+        return;
+      }
+      setTimeout(() => {
+        navigate("/login");
+      }, 2000);
+    });
+  }, []);
 
-    useEffect(() => {
-        callApi("GET", "/api/home", null)
-            .then((res) => {
-                if (res.ok) {
-                    setAuth(true)
-                    return
-                }
-                setTimeout(() => {
-                    navigate("/login")
-                }, 2000)
-            })
-    }, [])
-
-    return (
-        <Theme>
-            <Container height={100}>
-                {auth
-                    ?
-                    <>
-                        <Container width={100} heightKeyword="fit-content" justifyContent="flex-end"
-                                   alignItems="flex-end">
-                            <StyledThemeSwitcher onClick={() => {
-                                navigate("/theme")
-                            }}><Button primary>settings</Button></StyledThemeSwitcher>
-                        </Container>
-                        <Container justifyContent="space-evenly" height={100} widthMobile={75} widthTablet={60}
-                                   widthLaptop={45} widthDesktop={25}>
-                            <StyledHeader>
-                                <h1>Spellcaster</h1>
-                            </StyledHeader>
-                            <Container width={100}>
-                                <LeaderboardView/>
-                            </Container>
-                            <Container width={100}>
-                                <ButtonWrapper>
-                                    <CodeInput placeholder="Enter code..."></CodeInput>
-                                </ButtonWrapper>
-                                <ButtonWrapper>
-                                    <Button primary medium>Search For Game</Button>
-                                </ButtonWrapper>
-                            </Container>
-                        </Container>
-                    </>
-                    :
-                    <Container justifyContent="center" alignItems="center" height={100}>
-                        <p>You weren't authorized</p>
-                    </Container>
-                }
+  return (
+    <Theme>
```

Comment:
Look into usages of styled components theming in order to centralize layout color storage.

https://styled-components.com/docs/advanced#theming